requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 covdefaults
 coverage
 pytest
+ruamel.yaml
```

Comment:
this is not correct

Reply:
Deleted.

I was unable to run tests without this package.

Reply:
you probably have extremely old setuptools / pip / tox / virtualenv-- upgrade those

Reply:
Sure, looks like I misconfigured something.